src/gsi/crtm_interface.f90

```diff
@@ -859,10 +864,9 @@ subroutine init_crtm(init_pass,mype_diaghdr,mype,nchanl,nreal,isis,obstype,radmo
     endif ! nvege_type
  endif ! regional or IGBP
     
-! Initial GFDL saturation water vapor pressure tables
+! Initial GFDL saturation water vapor pressure tables 
   if (use_gfdl_qsat) then
-     if (n_actual_aerosols_wk>0 .or. n_clouds_fwd_wk>0 .and. imp_physics==11) then
-
+      if (n_actual_aerosols_wk>0 .or. n_clouds_fwd_wk>0 .and. imp_physics==11) then
```

Comment:
How many number of  spaces each level of indentation?

Reply:
The [GSI Code Standards Wiki](https://github.com/NOAA-EMC/GSI/wiki/GSI---Code-Standards#free-format-code) states that 3 spaces are used for indentation.

Reply:
Interesting. Any reason to choose 3?

Reply:
The choice of three traces back to our GSI code manager.  emacs in F90 mode defaults to an indentation of 3. There's nothing special about 3.  I see other standards use an indentation of 2 spaces.  The key is to use consistent indentation throughout a source code file.

Reply:
Thanks @RussTreadon-NOAA